test/sqllogictest/pg_get_constraintdef.slt

```diff
@@ -0,0 +1,27 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+#
+# This file is derived from the logic test suite in CockroachDB. The
+# original file was retrieved on June 10, 2019 from:
+#
+# The original source code is subject to the terms of the Apache
+# 2.0 license, a copy of which can be found in the LICENSE file at the
+# root of this repository.
+
+query error pg_get_constraintdef not yet supported
+SELECT pg_get_constraintdef(pg_constraint.oid) FROM pg_constraint
```

Comment:
Hmm, this is unfortunately not what we need. We need a query with this to successfully run, just that it won't ever have any rows so will happen to never call this function. We might need to error at a deeper level.

Reply:
the previous version of this commit had that behavior, with UnaryFunc::NotImplemented. does that work?

Reply:
I don't think so, as far as I can tell `NullaryFunc`'s must be infallible